### ISSUE TITLE
"MoDS Volume 2" demo not working on last build

### ISSUE THREAD

**Issue by tuzepoito** [TASEmulators/desmume#487]
Version: Dev 0.0.0.894 x64
OS: Windows 8.1

Issue: The [MoDS Volume 2](https://www.pouet.net/prod.php?which=78482) demo does not launch with the latest development version. No sound, no graphics (black screen)

It is to note that the demo does launch on the latest stable release (0.9.11 x64)

**Comment by zeromus** [comment]
we have to assume it's the same as this until proven otherwise: #486

**Comment by rogerman** [comment]
It does work and has always worked. See for yourself.
![MoDSVolume2](https://user-images.githubusercontent.com/6055122/141201560-fd3091e3-5190-4c6e-bd6c-9feed566a19f.png)

The issue here is that MoDS Volume 2, like many other homebrew ROMs, tend to be more sensitive to timing than other kinds of ROMs. It is true that MoDS Volume 2 will fail to start when using the Dynamic Recompiler for the CPU emulation, but setting the CPU emulation to Interpreter will allow MoDS Volume 2 to work. If for some reason you want to continue using Dynamic Recompiler, then set its Block Size to 4.

As a general rule, you should always revert the CPU emulation to Interpreter with Advanced Bus-Level Timing, which are DeSmuME's default emulation options, before  claiming that a ROM doesn't work. The default options are set that way to ensure compatibility, and any deviation is risking compatibility issues, like the one you are describing.

Closing, due to user error.

**Comment by tuzepoito** [comment]
For information, I did use CPU Interpreter option. After some testing, I can confirm it does launch ; booting time seems slow, though.
You see, on my setup, the demo takes around 15 seconds to boot until sound and graphics appear on the screen - which is why I assumed the demo did not launch. Also, it takes around 10 seconds to change to another track.
By comparison, launching the demo on v0.9.11 with default settings, boots immediately, and changing a track takes a few seconds.
It is to note that I get these results on both a Windows 8.1 laptop and a Windows 10 desktop. I notice you are using OS X.

By the way I found something interesting, forcing "PassME" into the GBA slot makes the ROM boot immediately, with no slow boot issue.
I got this result by noticing that launching "MoDS Volume 1" auto-selects PassME into the GBA slot - which is not the case with MoDS Volume 2.
